lib/ui/compositing.dart

```diff
@@ -307,8 +314,10 @@ class SceneBuilder extends NativeFieldWrapperClass1 {
     return layer;
   }
 
-  void _pushTransform(EngineLayer layer, Float64List matrix4, EngineLayer? oldLayer)
-      native 'SceneBuilder_pushTransform';
+  @FfiNative<Void Function(Pointer<Void>, Handle, Handle, Handle)>(
+      'SceneBuilder::pushTransformHandle')
```

Comment:
Here and elsewhere: this formatting does not follow the Flutter style guide. Please do not use dartfmt in this repo.  Lines like this should not be broken unless they greatly exceed 120 cols. The method below should definitely not be broken onto two lines.

Reply:
Prefer 80, but the prose under that says to violate that if it makes it more readable. Dartfmt with a 120 length is often closer to what we want.